### PR TITLE
Fix: #20 Add Dependabot Config

### LINKED ISSUE #20: Create a dependabot configuration
Create a dependabot.yml configuration to check if any package is outdated.

More about dependabot configuration can be read [here](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates).

### PR DESCRIPTION
**Pull request by dr5hn**
# This PR includes:
- Fix #20 - Create Dependabot configuration

### CODE REVIEW

File: .github/workflows/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    allow:
+      - dependency-type: "direct"
+    reviewers:
+      - "Tomburgs"
+    labels:
+      - "npm"
```
Comment (Tomburgs) [inline .github/workflows/dependabot.yml:12]: I don't think we need an npm label.

File: .github/workflows/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    allow:
+      - dependency-type: "direct"
+    reviewers:
+      - "Tomburgs"
+    labels:
```
Comment (Tomburgs) [inline .github/workflows/dependabot.yml:11]: Actually, it seems like dependabot will [add all the correct labels automatically](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates#labels).
I think we should skip this option.

File: .github/workflows/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    allow:
+      - dependency-type: "direct"
```
Comment (Tomburgs) [inline .github/workflows/dependabot.yml:8]: Smart! 🎉

**Review by Tomburgs — CHANGES_REQUESTED** [review]
Overall looks really good, but please have a look at the comments.

File: .github/workflows/dependabot.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```
[the remark this answers is not in the document]
Comment (dr5hn) [reply .github/workflows/dependabot.yml:6]: @Tomburgs You'll get annoyed. If i set it to Daily.

File: .github/workflows/dependabot.yml (hunk 1 above)
Comment (dr5hn) [reply .github/workflows/dependabot.yml:12]: @Tomburgs Ok Done.

File: .github/workflows/dependabot.yml (hunk 2 above)
Comment (dr5hn) [reply .github/workflows/dependabot.yml:11]: @Tomburgs Ok Done.

**Comment by dr5hn** [comment]
@Tomburgs Changes Done. I set as "Daily". Let me know if you want it "Weekly"

**Review by Tomburgs — APPROVED** [review]
Thank you, LGTM!

**Pull request merged**